eng/tools/spec-gen-sdk-runner/src/types.ts

```diff
@@ -34,9 +34,29 @@ export type VsoLogs = Map<
  * Represents the result of the spec-gen-sdk generation process.
  */
 export interface SpecGenSdkArtifactInfo {
-  managementPlane: boolean;
-  dataPlane: boolean;
-  language?: string;
+  language: string;
+  result: string;
```

Comment:
Is result a string or a bool?

Reply:
It's a string with values: ["succeeded", "failed" ...]